### DISCUSSION TITLE
Grain Families

### DISCUSSION THREAD

**Discussion by oMuransky** [mtex-toolbox/mtex#2043] [Ask Anything] [answered]
Hi, 

Is there a way to quickly find which grains in an EBSD map are aligned within 10 degrees with their specific plane normals in the specimen's X, Y, and Z directions? For example, I would like to find which grains are oriented within 10 degrees with their [100] plane normals in the X, Y, and Z sample directions. The idea is then to group the grains with similar orientations into grain families — something like the schematic below shown on a (100) pole figure.

Many thanks,
Ondrej

![268438280-632a2130-f6d3-430d-b39a-919aaec27d26](https://github.com/mtex-toolbox/mtex/assets/28768162/09f15ea4-057b-4067-bbb5-0af526a4e30e)

**Comment by kilir** [comment, marked as answer]
Hi,
I guess it's this:

```
mtexdata titanium
grains = ebsd.calcGrains;

% grains with 001 with 20 degree to x

% option 1:
cond = angle(inv(grains.meanOrientation) * vector3d(1,0,0), ...
             Miller(0,0,1,ebsd('t').CS)) < 20*degree;

plot(grains(cond))


%option 2:
cond = any(angle(grains.meanOrientation * symmetrise(Miller(0,0,1,ebsd('t').CS)), ...
            vector3d(1,0,0)) < 20*degree,2);
nextAxis
plot(grains(cond))

```
Cheers,
Rüdiger

**Comment by oMuransky** [comment]
Thank you, Rüdiger. That's exactly what I needed.
